ZenPacks/zenoss/WBEM/modeler/wbem.py

```diff
@@ -25,31 +25,54 @@
 set to succesfully pull data.
 
 """
+import itertools
 
 from twisted.internet import ssl, reactor
 from twisted.internet.defer import DeferredList, CancelledError
 
 from Products.DataCollector.plugins.CollectorPlugin import PythonPlugin
 
-from ZenPacks.zenoss.WBEM.utils import addLocalLibPath, result_errmsg
+from ZenPacks.zenoss.WBEM.utils import (
+    addLocalLibPath,
+    result_errmsg,
+    create_connection,
+)
 
 from ZenPacks.zenoss.WBEM.patches import (
     EnumerateClassNames,
     EnumerateClasses,
     EnumerateInstanceNames,
-    EnumerateInstances
+    EnumerateInstances,
+    OpenEnumerateInstances,
+    PullInstances,
 )
 
 addLocalLibPath()
 
 
+def getEnumerateInstances(creds, classname, namespace='root/cimv2', **kwargs):
```

Comment:
Please add a docstring to the function which explains what is the 'root/cimv2' namespace.
According to the module style, you have to use the following name for the function: `get_enumerate_instances`.